tools/travis/build.sh

```diff
@@ -49,6 +49,7 @@ TERM=dumb ./gradlew \
 :core:swift51Action:distDocker \
 :core:swift53Action:distDocker \
 :core:swift54Action:distDocker \
+:core:swift57Action:distDocker \
```

Comment:
The corresponding addition should be made after this line now:  https://github.com/apache/openwhisk-runtime-swift/blob/master/.github/workflows/ci.yaml#L103

Reply:
https://github.com/apache/openwhisk-runtime-swift/pull/153/commits/0707e20ec1070a6231135bf129de81b56bb91f49